MAINTAINERS.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Current Maintainers
+
+| Maintainer     | GitHub ID                                    | Affiliation                 |
+|----------------|----------------------------------------------|-----------------------------|
+| Jeff Zemerick  | [jzonthemtn](https://github.com/jzonthemtn)  | OpenSource Connections      |
+| Jason Robinson | [rasonj](https://github.com/rasonj)          | OpenSource Connections      |
+| Eric Pugh      | [epugh](https://github.com/epugh) | OpenSource Connections      |
```

Comment:
poor eric, he doesn't get the extract white space to be lined up :-).

Reply:
Now you're spacing matches!